src/box/lua/load_cfg.lua

```diff
@@ -336,6 +346,7 @@ local dynamic_cfg = {
     net_msg_max             = private.cfg_set_net_msg_max,
     sql_cache_size          = private.cfg_set_sql_cache_size,
     txn_timeout             = private.cfg_set_txn_timeout,
+    default_txn_isolation   = update_default_txn_isolation,
```

Comment:
Please add `default_txn_isolation` option to `dynamic_cfg_skip_at_load`or verify it in `box_check_config`:
https://github.com/tarantool/tarantool/blob/f1d750f036956873c79bf5de51f77b38da25ae0f/src/box/lua/load_cfg.lua#L284-L290